integrations/docker/images/chip-build/version

```diff
@@ -1 +1 @@
-0.4.4
+0.4.6
```

Comment:
you and I are both pushing to 0.4.6?

Reply:
No, I'm not pushing it. Just uploaded the PR.